src/components/Circles/Post.tsx

```diff
@@ -48,7 +69,7 @@ export const Post = ({ post }: PostProps) => {
           mode={'outlined'}
           onPress={() => {}} // TODO: Navigate to post details/comments page
         >
-          {post.replyCount} COMMENTS
+          {t('post-comments', { count: post.replyCount })}
```

Comment:
In tests I imagine this would render as `post-comments` based on the jest setup file. Is this what we want/expect?

Reply:
I don't think we have any test coverage aimed specifically at this so it won't cause any problems with testing but perhaps I should look into a way to fix this.

Reply:
On second thought, you are using this in your next PR for testing. Once you merge that in I will rebase that and fix any broken tests.

Reply:
Addressed by unmocking i18next